U/USRPHardwareDriver/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "USRPHardwareDriver"
+version = v"4.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/EttusResearch/uhd.git", "f633b497ff931c3c180d00148fc66e4feb8b8b6a")
+]
+
+dependencies = [
+    Dependency("boost_jll"; compat="=1.76.0"), # max gcc7
```

Comment:
Out of curiosity, why did you specify such a strict `boost_jll` compat version?

Reply:
Mentioned here: https://github.com/JuliaPackaging/Yggdrasil/issues/3024

Full version in the SONAME on linux.